virttest/qemu_vm.py

```diff
@@ -2153,8 +2160,8 @@ def sort_key(dev):
                 if arch.ARCH != 'ppc64le':
                     pci_id = ":".join(pci_id.split(":")[1:])
 
-                add_pcidevice(devices, pci_id, params=nic_params,
-                              device_driver=device_driver,
+                add_pcidevice(devices, pci_id, failover_pair_id,
+                              params=nic_params, device_driver=device_driver,
```

Comment:
It seems not necessary to specifically pass `failover_pair_id` since it is contained in `nic_params`, you might get it from `nic_params` inside `add_pcidevice()`.

Reply:
I have updated, and test it , it's passed. 

    -device pcie-root-port,id=pcie-root-port-3,port=0x3,addr=0x1.0x3,bus=pcie.0,chassis=4 \
    -device vfio-pci,host=3b:10.1,id=id_3b.10.1,failover_pair_id=net0,bus=pcie-root-port-3,addr=0x0 \
    -device pcie-root-port,id=pcie-root-port-4,port=0x4,addr=0x1.0x4,bus=pcie.0,chassis=5 \
    -device virtio-net-pci,mac=22:2b:62:bb:a9:82,id=net0,failover=on,netdev=idupDzi0,bus=pcie-root-port-4,addr=0x0  \
    -netdev tap,id=idupDzi0,fd=18  \